hack/test-extended/default/run.sh

```diff
@@ -32,21 +31,11 @@ cleanup() {
 	exit $out
 }
 
-# Check if we have ginkgo command
-set -e
-which ginkgo &>/dev/null || (echo 'Run: "go get github.com/onsi/ginkgo/ginkgo"' && exit 1)
-set +e
-
-# Compile the extended tests first to avoid waiting for OpenShift server to
-# start and fail sooner on compilation errors.
-echo "[INFO] Compiling test/extended package ..."
-GOPATH="${OS_ROOT}/Godeps/_workspace:${GOPATH}" \
-  go test -c ./test/extended -o ${OS_OUTPUT_BINPATH}/extended.test || exit 1
+require_ginkgo_or_die
+compile_extended_tests
 
 test_privileges
```

Comment:
+1 I was just writing docs for extended tests and found this name pretty confusing ;-) It is actually checking if the 'sudo' is there as well... Maybe we should split it to two functions and be more explicit.